CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: MIT
 
 cmake_minimum_required(VERSION 3.20)
+set(CMAKE_SYSTEM_VERSION 10.0.19041.0)
```

Comment:
Can you explain how this number was picked?  I thought Server 2019 was 17763

Reply:
This controls which version of the SDK is selected. Setting this to 10.0.22000.0 causes CMake to select the currently supported version of the SDK.